cpp/dolfinx/mesh/cell_types.cpp

```diff
@@ -120,25 +119,22 @@ mesh::get_sub_entities(CellType type, int dim0, int dim1)
         "mesh::get_sub_entities supports getting edges (d=1) at present.");
   }
   // TODO: get this data from basix
-  const static Eigen::Array<int, 1, 3, Eigen::RowMajor> triangle
-      = (Eigen::Array<int, 1, 3, Eigen::RowMajor>() << 0, 1, 2).finished();
-  const static Eigen::Array<int, 1, 4, Eigen::RowMajor> quadrilateral
-      = (Eigen::Array<int, 1, 4, Eigen::RowMajor>() << 0, 1, 2, 3).finished();
-  const static Eigen::Array<int, 4, 3, Eigen::RowMajor> tetrahedron
-      = (Eigen::Array<int, 4, 3, Eigen::RowMajor>() << 0, 1, 2, 0, 3, 4, 1, 3,
-         5, 2, 4, 5)
-            .finished();
-  const static Eigen::Array<int, 6, 4, Eigen::RowMajor> hexahedron
-      = (Eigen::Array<int, 6, 4, Eigen::RowMajor>() << 0, 1, 3, 5, 0, 2, 4, 8,
-         1, 2, 6, 9, 3, 4, 7, 10, 5, 6, 7, 11, 8, 9, 10, 11)
-            .finished();
-
+  dolfinx::common::array2d<int> triangle({{0, 1, 2}});
```

Comment:
Could this be just a `std::array<std::array>`? No need for dynamic memory.

Reply:
Yep,  but I think it would complicate the return type `std::array<std::array<int, N>, M>` with M, N known at compile time (auto would fail).
Maybe we could handle this in a different PR?